guides/back-end-projects.md

```diff
@@ -0,0 +1,104 @@
+---
+explains: The style guide to the way we organize our back-end projects
+---
+
+# How we use docker and docker-compose
+
+A lot of the software we make at City of Amsterdam uses docker containers.
+
+In general we follow the common community guidelines:
+- [Dockerfile best practices](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/)
+, including multi-stage builds
+
+Preferably use [docker-ce](https://www.docker.com/community-edition)
+and [docker-compose](https://docs.docker.com/compose/), most recent version
+
+.dockerignore files are used to exclude files that are not relevant to the build.
```

Comment:
It not very common to start a sentence with a dot, so if you want to, maybe emphasize it with backticks.